### DISCUSSION TITLE
Using bar Chart for past 24h Energy production showing last 24h

### DISCUSSION THREAD

**Discussion by Marcwa19197** [kalkih/mini-graph-card#1055] [Q&A]
Hello,

im using the following YAML:
```
type: custom:mini-graph-card
entities:
  - sensor.producedenergylifetime
name: Heutiger Solarertrag
hours_to_show: 168
align_state: center
decimals: 5
aggregate_func: max
group_by: date
show:
  graph: bar
```

I want to achieve the following:
sensor.producedenergylifetime is showing the lifetime produced energy. (Value increases from time to time)

- Showing a 7day Bar Chart with the daily produced energy in kWh with 5 decimals.
- Showing the last 24h as value directly on the entity (instead of the current value of sensor.producedenergylifetime).

With above code i see the 7day bar chart, which is great, but the "big value" is the current state value of the sensor and not the maximum value from the past 24h.

<img width="489" alt="image" src="https://github.com/kalkih/mini-graph-card/assets/5463753/70c73166-df2e-4c53-a871-047ec8950d5d">

So basically the value which should be displayed is the current state minus the summary of the last 24h.

Thanks for any advise!

**Comment by Marcwa19197** [comment]
no one? Or did i make something basically wrong using the card? :-)
